src/installer.js

```diff
@@ -56,6 +56,7 @@ class DebianInstaller extends common.ElectronInstaller {
    */
   copyApplication () {
     return super.copyApplication(src => src !== path.join(this.options.src, 'LICENSE'))
+      .then(() => this.updateSandboxHelperPermissions())
```

Comment:
So I'm not sure that `copyApplication` is actually the right place for this. It should rather happen when we copy the `electron` binary (and `resources`, `locales`, etc.). I'm not sure what the right function for that is though.

Reply:
I'm a little confused. `copyApplication` is where we copy the bundled Electron app to the correct folder in the staging directory.

Are you referring to when we extract the prebuilt Electron binary from the zip file retrieved from `github.com/electron/electron/releases`?

Reply:
yeah, this should happen when we extract the dist zip (which will contain chrome-sandbox with the wrong permissions), not when we copy the app's JS.

Reply:
I was wondering about this in Slack, but as @MarshallOfSound responded to me:

> You can't guarantee that the [bundled app that Electron Packager] outputs won't be moved between build machines before the [installer modules] run

In which case, putting the code here might be the safest option.

Reply:
The example I use here for a Good™️ CI setup is this --> https://circleci.com/workflow-run/ac96f5d9-2c62-4b1f-8ec3-5c7ba9964535

Packaging happens on a different machine to the building of the `.deb` and `.rpm` files.

Reply:
In that case, this should probably rather just be a separate step. It doesn't need to happen in sequence with `copyApplication`. I'll update the PR.

Reply:
I didn't add it to `contentFunctions` because `copyContents` runs everything in a `Promise.all`, so it's not guaranteed that `updateSandboxHelperPermissions` will be running after the application is copied over. I wrote `updateSandboxHelperPermissions` in the context of a copied over application.

Reply:
I need to add more documentation to `electron-installer-common`, I'll add it to my very large TODO list.